packages/salesforcedx-vscode-apex-replay-debugger/src/messages/i18n.ts

```diff
@@ -21,6 +21,8 @@ export const messages = {
     'Apex language server could not provide information about valid breakpoints.',
   up_to_five_checkpoints:
     'You have %s of the maximum 5 enabled checkpoints. Remove or disable checkpoints until 5 or fewer are active.',
+  no_enabled_checkpoints:
```

Comment:
We decided in our demo that we would like to keep the warning. @sbudhirajadoc Could you help us review the `no_enabled_checkpoints` message? Thanks!

Reply:
Can you change it to "You don't have any checkpoints enabled." 
Also, are they required to take a followup action that might require saying, "Enable a checkpoint and try again." ?

Reply:
Got it! Users are still able to run through tests without any checkpoints enabled, so **not** having an "Enable a checkpoint and try again" message makes more sense to me. @AnanyaJha @jag-sfdc Let us know what you think!

Reply:
We decided in daily sync that we should add "Enable a checkpoint and try again"! `i18n.ts` has been updated.